src/devices/Card/Ndef/TypeNameFormat.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// The format type
+    /// </summary>
+    public enum TypeNameFormat
```

Comment:
Should this TypeFormat? (looking at the description)

Reply:
It's more aligned with the official name which is Type Name Format (TNF). I'll adjust the description.